From 19b0d016871473e8cb717cae5d01f062b06b6643 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Robert-Andr=C3=A9=20Mauchin?= <zebob.m@gmail.com>
Date: Fri, 30 Nov 2018 19:03:27 +0100
Subject: [PATCH] Remove default shortcuts

---
 src/globalshortcuts/globalshortcuts.cpp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/globalshortcuts/globalshortcuts.cpp b/src/globalshortcuts/globalshortcuts.cpp
index f22f04f..07b4d31 100644
--- a/src/globalshortcuts/globalshortcuts.cpp
+++ b/src/globalshortcuts/globalshortcuts.cpp
@@ -51,11 +51,11 @@ GlobalShortcuts::GlobalShortcuts(QWidget *parent)
   // Create actions
   AddShortcut("play", tr("Play"), SIGNAL(Play()));
   AddShortcut("pause", tr("Pause"), SIGNAL(Pause()));
-  AddShortcut("play_pause", tr("Play/Pause"), SIGNAL(PlayPause()), QKeySequence(Qt::Key_MediaPlay));
-  AddShortcut("stop", tr("Stop"), SIGNAL(Stop()), QKeySequence(Qt::Key_MediaStop));
+  AddShortcut("play_pause", tr("Play/Pause"), SIGNAL(PlayPause()));
+  AddShortcut("stop", tr("Stop"), SIGNAL(Stop()));
   AddShortcut("stop_after", tr("Stop playing after current track"), SIGNAL(StopAfter()));
-  AddShortcut("next_track", tr("Next track"), SIGNAL(Next()), QKeySequence(Qt::Key_MediaNext));
-  AddShortcut("prev_track", tr("Previous track"), SIGNAL(Previous()), QKeySequence(Qt::Key_MediaPrevious));
+  AddShortcut("next_track", tr("Next track"), SIGNAL(Next()));
+  AddShortcut("prev_track", tr("Previous track"), SIGNAL(Previous()));
   AddShortcut("inc_volume", tr("Increase volume"), SIGNAL(IncVolume()));
   AddShortcut("dec_volume", tr("Decrease volume"), SIGNAL(DecVolume()));
   AddShortcut("mute", tr("Mute"), SIGNAL(Mute()));
-- 
2.19.1

