diff -up strawberry-0.6.3/src/CMakeLists.txt.udisks strawberry-0.6.3/src/CMakeLists.txt
--- strawberry-0.6.3/src/CMakeLists.txt.udisks	2019-08-08 23:14:43.948483881 +0200
+++ strawberry-0.6.3/src/CMakeLists.txt	2019-08-08 23:18:59.966764619 +0200
@@ -662,6 +662,9 @@ if(UNIX AND HAVE_DBUS)
   list(APPEND HEADERS ${CMAKE_CURRENT_BINARY_DIR}/dbus/avahientrygroup.h)
   list(APPEND SOURCES ${CMAKE_CURRENT_BINARY_DIR}/dbus/avahientrygroup.cpp)
 
+  set_source_files_properties(dbus/org.freedesktop.UDisks.xml
+     dbus/org.freedesktop.UDisks.Device.xml PROPERTIES NO_NAMESPACE TRUE)
+  
   if(HAVE_UDISKS2)
        set_source_files_properties(dbus/org.freedesktop.DBus.ObjectManager.xml
            PROPERTIES NO_NAMESPACE dbus/objectmanager INCLUDE dbus/metatypes.h)
