diff -up strawberry-0.3.3/CMakeLists.txt.use_qt5 strawberry-0.3.3/CMakeLists.txt
--- strawberry-0.3.3/CMakeLists.txt.use_qt5	2018-10-22 03:14:34.852939691 +0200
+++ strawberry-0.3.3/CMakeLists.txt	2018-10-22 03:16:11.515764262 +0200
@@ -211,9 +211,9 @@ endif()
 option(USE_SYSTEM_QTSINGLEAPPLICATION "Use system QtSingleApplication library" OFF)
 if(USE_SYSTEM_QTSINGLEAPPLICATION)
   message(STATUS "Using system QtSingleApplication library")
-  find_path(QTSINGLEAPPLICATION_INCLUDE_DIRS qtsingleapplication.h PATH_SUFFIXES QtSolutions)
-  find_library(QTSINGLEAPPLICATION_LIBRARIES QtSolutions_SingleApplication-2.6)
-  find_library(QTSINGLECOREAPPLICATION_LIBRARIES QtSolutions_SingleCoreApplication-2.6)
+  find_path(QTSINGLEAPPLICATION_INCLUDE_DIRS qtsingleapplication.h PATH_SUFFIXES qt5/QtSolutions)
+  find_library(QTSINGLEAPPLICATION_LIBRARIES Qt5Solutions_SingleApplication-2.6)
+  find_library(QTSINGLECOREAPPLICATION_LIBRARIES Qt5Solutions_SingleCoreApplication-2.6)
 else(USE_SYSTEM_QTSINGLEAPPLICATION)
   message(STATUS "Using builtin QtSingleApplication library")
   add_subdirectory(3rdparty/qtsingleapplication)
@@ -225,11 +225,11 @@ endif(USE_SYSTEM_QTSINGLEAPPLICATION)
 option(USE_SYSTEM_QXT "Use system Qxt library" OFF)
 if (USE_SYSTEM_QXT)
   message(STATUS "Using system Qxt library")
-  find_path(QXTCORE_INCLUDE_DIRS qxtglobal.h PATH_SUFFIXES QxtCore)
-  find_path(QXTGUI_INCLUDE_DIRS qxtglobalshortcut.h PATH_SUFFIXES QxtGui)
+  find_path(QXTCORE_INCLUDE_DIRS qxtglobal.h PATH_SUFFIXES qt5/QxtCore)
+  find_path(QXTGUI_INCLUDE_DIRS qxtglobalshortcut.h PATH_SUFFIXES qt5/QxtWidgets)
   set(QXT_INCLUDE_DIRS ${QXTCORE_INCLUDE_DIRS} ${QXTGUI_INCLUDE_DIRS})
   # We only need its header. We don't need to link to QxtCore.
-  find_library(QXT_LIBRARIES QxtGui)
+  find_library(QXT_LIBRARIES QxtWidgets-qt5)
 else (USE_SYSTEM_QXT)
   message(STATUS "Using builtin Qxt library")
   add_definitions(-DQXT_STATIC -DBUILD_QXT_GUI -DBUILD_QXT_CORE)
diff -up strawberry-0.3.3/src/CMakeLists.txt.use_qt5 strawberry-0.3.3/src/CMakeLists.txt
--- strawberry-0.3.3/src/CMakeLists.txt.use_qt5	2018-10-22 03:14:34.853939689 +0200
+++ strawberry-0.3.3/src/CMakeLists.txt	2018-10-22 03:16:36.357719178 +0200
@@ -880,6 +880,7 @@ target_link_libraries(strawberry_lib
   ${QT_LIBRARIES}
   ${CHROMAPRINT_LIBRARIES}
   ${QTSINGLEAPPLICATION_LIBRARIES}
+  ${QTSINGLECOREAPPLICATION_LIBRARIES}
   ${CMAKE_THREAD_LIBS_INIT}
   ${SQLITE_LIBRARIES}
   ${QOCOA_LIBRARIES}
